news/templates/news/article_detail.html

```diff
@@ -25,6 +25,12 @@
 					<h1><a href="#">{{ article.title }}</a></h1>
 					{{ article.content|safe }}
 				</div>
+				<p>This artickle habe {{likes}} tricks</p>
+				<p>{{liked}}</p>
+				<form action="like" method="POST">
```

Comment:
better change action to  django's {% url %} (check other template files for examples)